src/mol-state/tree/transient.ts

```diff
@@ -203,6 +203,25 @@ class TransientTree implements StateTree {
         return true;
     }
 
+    setDependsOn(ref: StateTransform.Ref, dependsOn: string | string[] | undefined) {
```

Comment:
What is the purpose of this function?

Reply:
Some of the primitives depend on structure nodes being resolved first.